saleor/warehouse/models.py

```diff
@@ -139,6 +147,15 @@ def decrease_stock(self, quantity: int, commit: bool = True):
         if commit:
             self.save(update_fields=["quantity"])
 
+    def available_quantity(self):
```

Comment:
IMO we should calculate it explicitly instead of using this function. 
Someone can accidently generate a lot of database queries when using this function.